packages/e2e-tests/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+### New features
```

Comment:
I'm not sure about the CHANGELOG policy in the `e2e-tests` package — it looks like only new features and breaking changes are listed (and not new tests)